python/mujoco/specs_test.py

```diff
@@ -433,7 +435,9 @@ def test_kwarg(self):
     )
 
   def test_load_xml(self):
-    filename = '../../test/testdata/model.xml'
+    filename = os.path.join(
+      epath.resource_path("mujoco"), "testdata", "model.xml"
+    )
```

Comment:
Would this work?
```suggestion
        file_path = epath.Path(__file__).parents[2] / "test" / "testdata" / "model.xml"
```

Reply:
Then, below, you'd do:
```
spec1 = mujoco.MjSpec.from_file(file_path.as_posix())
```

Reply:
I am not sure. I guess the problem is that the `test/testdata` folder is never installed, so no matter which relative path you pass, if you install `mujoco` via for example a wheel, the needed file `model.xml` is not found anywhere.